exo-clean/.travis.yml

```diff
@@ -0,0 +1,31 @@
+language: node_js
+
+sudo: required
+dist: trusty
+
+services:
+  - docker
+
+node_js:
+  - "6"
+  - "7"
+
+
+before_install:
+  - npm install exosphere-shared
+  - docker pull originate/exocom:0.15.1
+  - docker pull exospheredev/web
+  - docker pull exospheredev/web-server
+  - docker pull exospheredev/mongo-service
+  - docker pull exospheredev/crasher
+  - docker pull exospheredev/runner
```

Comment:
I think these `docker pull` commands can be removed, we won't be needing those images for this sub-directory's tests.